LICENSE

```diff
@@ -1,21 +1,2 @@
-MIT License
```

Comment:
I think this file is the correct place to list the license, I think the LICENSE.md can be removed from the repo.